### PR TITLE
[INTERNAL] ESSupport.md: Add a section about reserved variable names

### PR DESCRIPTION
**Pull request by flovogt**
List the reserved variable names inside Template Literal expressions.

### CODE REVIEW

File: docs/pages/ESSupport.md [hunk 1]
```diff
@@ -167,6 +167,30 @@ A library is typically initialized via an accompanying `library.js`. Within that
     });
     ```
 
+#### Reserved Variable Names in a Template Literal
+
+Template Literals with an expression requires to relinquish following variable names:
```
Comment (RandomByte) [inline docs/pages/ESSupport.md:172]: From the top of my head, I did not know the meaning of `relinquish` 😄 

How about:

```suggestion
Template Literals with an embedded expression must not use the following variable names:
```

**Review by RandomByte — CHANGES_REQUESTED** [review]

File: docs/pages/ESSupport.md (hunk 1 above)
Comment (matz3) [reply docs/pages/ESSupport.md:172]: The reason for this limitation should be explained. We could even suggest to use https://eslint.org/docs/latest/rules/template-curly-spacing with option "always" to mitigate this. Our placeholder replacement does not allow spaces.

Comment (flovogt) [reply docs/pages/ESSupport.md]: Tackled both comments.

File: docs/pages/ESSupport.md [hunk 2]
```diff
@@ -167,6 +167,31 @@ A library is typically initialized via an accompanying `library.js`. Within that
     });
     ```
 
+#### Reserved Variable Names in a Template Literal
+
+While UI5 Tooling performs a build placeholders are replaced with a values offered by the build. For example `${version}` is replaced with the actual version defined in the package.json of the project. Thereore it is required to not use any **Template Literal** where any expression contains variable with folloing names:
```
Comment (matz3) [inline docs/pages/ESSupport.md:172]:
```suggestion
While UI5 Tooling performs a build placeholders are replaced with a values offered by the build. For example `${version}` is replaced with the actual version defined in the package.json of the project. Therefore it is required to not use any **Template Literal** where any expression contains variable with following names:
```

**Review by matz3 — APPROVED** [review]

**Pull request merged**